src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --maincolor: #00acdc;
+  --header-height: 60px;
+  --header-margin: 50px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. If 'Roboto' fails to load, the browser will use the generic 'sans-serif' font. It's a good practice to specify at least one more specific fallback font before the generic font family.